### ISSUE TITLE
Since this repository isn't being developed, I'm forking it, to keep it developed.

### ISSUE THREAD

**Issue by OrdoFlammae** [littleosbook/littleosbook#95]
The title basically says it all. I see a bunch of valid, and indeed, useful pull requests, but the book isn't being developed at all, which is a shame. I really liked it. Therefore, I'm going to fork it, and keep the fork developed, so as to try to keep this book alive. If the creators of the book want to start keeping it alive, they can close this issue at any point. When this issue is closed, I will probably remove the repositories I forked. Repository at https://github.com/OrdoFlammae/littleosbook-src. Book itself is at https://ordoflammae.github.io/littleosbook/.

**Comment by Phlosioneer** [comment]
We should probably start copying issues over. I kinda wanted to do this but I'm to busy to directly maintain it.

Does the fork have the PR's already merged?

**Comment by OrdoFlammae** [comment]
No, I couldn't figure out how to copy the pull requests while keeping the credit for the people who originated them. If you know how, could you tell me, or better, link me to an explanation page on the GitHub help site (if there is one)?

EDIT: Never mind, I just figured it out. Might take a while to get them all though.

**Comment by Jont828** [comment]
Are you taking issues/pull requests on the forked book? I was thinking about adding something about how to install the prerequisites on a non-Ubuntu system since some packages like `bochs-sdl` are not available on places like the AUR. Also, I thought it would be good to have a complete, running copy of the whole OS following the book's code so that people can make sure they're setting everything up correctly.

**Comment by OrdoFlammae** [comment]
Yes, I am taking issues/pull requests on the forked repository. Right now, I'm working on transferring it to GRUB 2.

**Comment by HimoriK** [comment]
> No, I couldn't figure out how to copy the pull requests while keeping the credit for the people who originated them. If you know how, could you tell me, or better, link me to an explanation page on the GitHub help site (if there is one)?
> 
> EDIT: Never mind, I just figured it out. Might take a while to get them all though.

@OrdoFlammae For the sake of science, how do you do this to revive deserted projects?

**Comment by OrdoFlammae** [comment]
@HimoriK, if you're still interested in my methods, I merely created pull requests into my repository based on the original branches that the pull requests into this repository were based on. This appears to keep the authors of the PRs intact without causing too much trouble. It helped that there were not a whole lot of large changes pending review. For example, take https://github.com/OrdoFlammae/littleosbook/pull/1. I created the PR, but it was based off the original author's branch and work, so credit was (ideally) still attached to where it belonged.
